attributes/default.rb

```diff
@@ -260,3 +260,39 @@
 default['ndb']['mysqld_exporter']['home']                   = "#{node['ndb']['dir']}/mysqld_exporter-#{node['ndb']['mysqld_exporter']['version']}.linux-amd64"
 default['ndb']['mysqld_exporter']['base_dir']               = "#{node['ndb']['dir']}/mysqld_exporter"
 default['ndb']['mysqld']['metrics_port']                    = "9104"
+
+
+# Rondb Rest API Server Configurations
+default['ndb']['rdrs']['version']                                      = "0.1.0"
+default['ndb']['rdrs']['internal']['buffer_size']                      = "327680"
+default['ndb']['rdrs']['internal']['pre_allocated_buffers']            = "1024"
+default['ndb']['rdrs']['internal']['go_max_procs']                     = "-1"
```

Comment:
Can you leave a comment that n < 1 defaults to number of CPUs?

Reply:
fixed